Maths/test/FindTrianglePerimeter.test.js

```diff
@@ -0,0 +1,19 @@
+import { findTrianglePerimeter } from '../FindTrianglePerimeter'
+
+describe('findTrianglePerimeter', () => {
+  it('should return the perimeter of a triangle', () => {
```

Comment:
redundant `it` block; does a simple sum of three values really need 4 tests?

Reply:
Sorry new to git. Ya might be too many. Maybe I will find another algorithms that is not trivial....